### DISCUSSION TITLE
Is Soak Testing with Artillery possible? What is the max. duration we can define? Can I have 12 hours duration time?

### DISCUSSION THREAD

**Discussion by sultanzarif** [artilleryio/artillery#1572] [Artillery]
Hi all,

I would like to do soak / endurance testing of our core systems with Artillery. I want to test the system under an average load for an extended period of time by executing a subset of processes, planning to have 12 hours extended time of testing period. Is it possible with Artillery?

Thanks for your replies in advance!

**Comment by hassy** [comment]
Hi @sultanzarif 👋 There is no hard limit on the duration of tests with Artillery. I know of users that routinely run tests for 9-12 hours.

**Reply by sultanzarif** [reply]
That is great to know! Thank you @hassy for quick reply!
